tests/detect-vlan-id/README.md

```diff
@@ -0,0 +1,6 @@
+Test for checking the working of vlan.id keyword.
```

Comment:
Commit message should rather be titled `detect:` rather than `test:` because almost every commit in SV is a test...